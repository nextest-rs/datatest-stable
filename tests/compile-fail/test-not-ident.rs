datatest_stable::harness! {
    { "xyz" = foo }
}
