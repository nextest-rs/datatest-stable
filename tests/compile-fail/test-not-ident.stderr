error: expected `test`, found non-identifier token: (rest: "xyz" = foo,)
 --> tests/compile-fail/test-not-ident.rs:1:1
  |
1 | / datatest_stable::harness! {
2 | |     { "xyz" = foo }
3 | | }
  | |_^
  |
  = note: this error originates in the macro `$crate::harness_collect` which comes from the expansion of the macro `datatest_stable::harness` (in Nightly builds, run with -Z macro-backtrace for more info)
